deepforest/cascade.py

```diff
@@ -1062,3 +1290,139 @@ def predict(self, X):
         proba = self.predict_proba(X)
         y = self._decode_class_labels(np.argmax(proba, axis=1))
         return y
+
+
+@deepforest_model_doc(
+    """Implementation of the deep forest for regression.""", "regressor_model"
+)
+class CascadeForestRegressor(BaseCascadeForest, RegressorMixin):
+    def __init__(
+        self,
+        n_bins=255,
+        bin_subsample=2e5,
+        bin_type="percentile",
+        max_layers=20,
+        n_estimators=2,
+        n_trees=100,
+        max_depth=None,
+        min_samples_leaf=1,
+        use_predictor=False,
+        predictor="forest",
+        predictor_kwargs={},
+        n_tolerant_rounds=2,
+        delta=1e-5,
+        partial_mode=False,
+        n_jobs=None,
+        random_state=None,
+        verbose=1,
+    ):
+        super().__init__(
+            n_bins=n_bins,
+            bin_subsample=bin_subsample,
+            bin_type=bin_type,
+            max_layers=max_layers,
+            n_estimators=n_estimators,
+            n_trees=n_trees,
+            max_depth=max_depth,
+            min_samples_leaf=min_samples_leaf,
+            use_predictor=use_predictor,
+            predictor=predictor,
+            predictor_kwargs=predictor_kwargs,
+            n_tolerant_rounds=n_tolerant_rounds,
+            delta=delta,
+            partial_mode=partial_mode,
+            n_jobs=n_jobs,
+            random_state=random_state,
+            verbose=verbose,
+        )
+
+    def _repr_performance(self, pivot):
+        msg = "Val Acc = {:.3f}"
+        return msg.format(pivot)
+
+    @deepforest_model_doc(
+        """Build a deep forest using the training data.""", "regressor_fit"
+    )
+    def fit(self, X, y, sample_weight=None):
+        super().fit(X, y, sample_weight)
+
+    def predict(self, X):
+        """
+        Predict class for X.
```

Comment:
The docstrings for regressor predict should be `Predict regression target for X.`